resolvconf/dns/resolvconf.go

```diff
@@ -4,6 +4,9 @@ import (
 	"regexp"
 )
 
+// DefaultIPv4Servers is the list of default ipv4 nameservers
+var DefaultIPv4Servers = []string{"8.8.8.8", "8.8.4.4"}
```

Comment:
Usage of defaultIPv4Dns is a bit different. so its defined as ```"nameserver 8.8.8.8", "nameserver 8.8.4.4"```; I just wanted the IPs here. That is why I added a new variable. But let me either move it to resolvconf package or use defaultIPv4Dns with some string ops to get the IPs.